server/app/controllers/DocumentCtrl.js

```diff
@@ -0,0 +1,238 @@
+import db from '../models/index';
+import * as auth from '../helpers/AuthHelper';
+
+/**
+ * createDocument
+ * Create a new document
+ * @param {Object} req The Request object
+ * @param {Object} res The Response from the server
+ * @return {undefined}
+ */
+const createDocument = (req, res) => {
+  db.Documents
+    .create({
+      title: req.body.title,
+      content: req.body.content,
+      owner: req.user.userId,
+      access: req.body.access || 'public'
```

Comment:
Thanks @andela-uenabulele 

I actually have this in my model and this is unnecessary duplication.